src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +44,35 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+        final String pattern = "dd.MM.yyyy";
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(pattern);
+        LocalDate to = LocalDate.parse(dateTo, formatter);
+        LocalDate from = LocalDate.parse(dateFrom, formatter);
+        if (to.compareTo(from) < 0) {
+            throw new IllegalDateParametersException();
```

Comment:
try to use isAfter() or isBefore() those are more relevant methods

Reply:
done